bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorageTest.java

```diff
@@ -241,16 +242,20 @@ public void testBookieCompaction() throws Exception {
     @Test
     public void doubleDirectory() throws Exception {
         int gcWaitTime = 1000;
+        File firstDir = new File("dir1");
```

Comment:
nit: a better approach would be create the subdir under the tmpDir initialized in this test suite.

```
File firstDir = new File(tmpDir, "dir1");
File secondDir = new File(tmpDir, "dir2");
```